### ISSUE TITLE
Preview View - Graph Preview Filters - Graph Value Filter

### ISSUE THREAD

**Issue by iskandarbasman** [Cacti/cacti#4774]
# Feature Request

## Is your feature request related to a problem? Please describe

I have many Interface graphs and this case use lets use graph template "Interface- Errors/Discards" as an example.
Currently there is no way to filter graphs that based on certain graph values.

## Describe the solution you'd like

It would be great to have ability to Filter say  
Item = "Errors In" 
Operator = ">"
Value =  "0"
Out of the hundreds or thousands of graphs that I currently have, you will only see those graphs that actual have error values above 0.

## Describe alternatives you've considered

Solution is above which is a new graph filter option based on graph value.

## Additional context

NA.

**Comment by TheWitness** [comment]
Yea, this is a duplicate of another 'sort by' feature request.  It's getting closer, and at the same time further away from completion, but it's still in my vision of the future.  The key thing to do is to revise the Data Source Statistics feature to properly scale and to add a few extra's before introducing this, and the other feature.

**Comment by TheWitness** [comment]
When I say 'scale', it's doing pretty good now that it's done in parallel, but I need to see how it behaves with 2M+ questions (data sources).

**Comment by iskandarbasman** [comment]
Okay.. to know there is this is in the roadmap. Thanks @TheWitness

**Comment by TheWitness** [comment]
I lack not the inspiration to do great things, but only the time to achieve them.

**Comment by TheWitness** [comment]
So this has been mostly fixed now.  I will make some additional tweaks in the coming weeks.

**Comment by TheWitness** [comment]
This is related to #2572 which I marked complete, but for which I'd like to make a few more enhancements.
